### PR TITLE
Fix misc bugs and issues

### PR DESCRIPTION
**Pull request by FabienRoger**

### CODE REVIEW

**Comment by Butanium** [comment]
Won't that break import using `elk.` ? Like 
```py
# generation_main.py
from elk.utils_generation.generation import create_records, create_hiddenstates
```

**Comment by FabienRoger** [comment]
> Won't that break imports using `elk.` ? Like
> 
> ```python
> # generation_main.py
> from elk.utils_generation.generation import create_records, create_hiddenstates
> ```

To `cd` in elk? No, because elk is pip installed module. Using `elk.` works anywhere.

File: elk/train.py [hunk 1]
```diff
@@ -54,6 +57,7 @@ def train(args):
 
     # save models
     # TODO: use better filename for the pkls, so they don't get overwritten
+    Path(args.trained_models_path).mkdir(parents=True, exist_ok=True)
```
Comment (norabelrose) [inline elk/train.py:60]: nit: `args.trained_models_path` is already a `Path` so we don't have to make it one here

**Review by norabelrose — APPROVED** [review]
LGTM, I have one little code style nit about line 60 of `train.py` but that could be changed in a later PR or something

**Pull request merged**